plugins/font/plugin.js

```diff
@@ -39,7 +39,7 @@
 			defaultValue: 'cke-default',
 			allowedContent: style,
 			requiredContent: style,
-			contentTransformations: [
+			contentTransformations: styleDefinition.element === 'span' ? [
```

Comment:
This fix seems too specific. What if I change default style element to e.g. `em` or `my-font`?

Additionally the transformation itself should change `font` to `styleDefinition.element`, not hardcoded `span` (however this could be moved to separate issue).

Reply:
@Comandeer from what I see this coherent with what we have already. Since 4.6.0 we have `contentTransformation` which applies `span`.
It sounds like this case should be covered by entire separate issue. WDYT?

https://github.com/ckeditor/ckeditor-dev/blob/6eb7105d2499950b6da4b63a04851b412af60086/plugins/font/plugin.js#L42-L83

Reply:
Yes, it could be extracted to separate issue.

Reply:
Extracted here: #2315